_posts/2021-02-19-26-rln-relay.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+name:  "Integrating rate limiting nullifiers into Waku Relay to enable spam-protection."
+title:  "Integrating rate limiting nullifiers into Waku Relay to enable spam-protection."
+date:   2021-02-25 12:00:00 +0800
+author: staheri14
+published: true
+permalink: /waku-v2-rln-relay
+categories: research
+summary: A simplified explanation of how Waku RLN-Relay mitigates the spamming issue in a p2p manner.
+image: 
+discuss: 
+---
+
+# Introduction
+
+This blog is going to give you an overview of how we enabled spam protection in waku relay through rate-limiting nullifiers or rln for short.
```

Comment:
Nitpick but it's more of a research log, less of a blog per se, i.e. I'd just call it "post".

Also capitalization on waku relay, rln etc

Reply:
How we enabled sounds like it is already enabled, as oppose to "how this can be achieved" It is still very experimental and far from lvie